static/app/components/replays/breadcrumbs/replayComparisonModal.tsx

```diff
@@ -25,40 +28,37 @@ export default function ReplayComparisonModal({
   replay,
   rightOffsetMs,
 }: Props) {
+  const integrationsContext = useAsyncSDKIntegrationStore();
+
   return (
     <OrganizationContext.Provider value={organization}>
-      <Header closeButton>
-        <ModalHeader>
-          <h4>
-            Hydration Error
-            <FeatureBadge type="beta" />
-          </h4>
-          <GithubFeedbackButton
-            href="https://github.com/getsentry/sentry/discussions/62097"
-            label={t('Discussion')}
-            title={null}
-            analyticsEventKey="replay.details-hydration-discussion-clicked"
-            analyticsEventName="Replay Details Hydration Discussion Clicked"
-            priority="primary"
+      <AsyncSDKIntegrationContextProvider outerContext={integrationsContext}>
+        <Header closeButton>
+          <ModalHeader>
+            <h4>
+              Hydration Error!!
```

Comment:
exclamations just for fun?🤓

Reply:
This was from a test to make sure I had the right component lol